shared/gh/js/controllers/gh.api.util.js

```diff
@@ -437,6 +437,47 @@ define(['exports', 'moment', 'bootstrap-notify'], function(exports, moment) {
     };
 
     /**
+     * Order the terms and split up the out of term events
+     *
+     * @param  {terms}     terms    The terms that should be reordered
+     * @return {Object}             Object containing the reordered terms and events
+     */
+    var orderEventsByTerm = exports.orderEventsByTerm = function(terms) {
```

Comment:
The function name doesn't describe properly what it does. Do we need this though? It seems to be splitting out OT events into their own term, but that's what should be happening in `splitEventsByTerm` so maybe we can move this logic in there.